src/main/java/com/openlattice/chronicle/services/ChronicleServiceImpl.java

```diff
@@ -538,7 +537,7 @@ public Integer updateAppsUsageAssociationData(
                             .iterator()
                             .next()
                             .toString()
-                            .startsWith( currentDate )
+                            .startsWith( date )
```

Comment:
Since this is a URL, nothing is stopping me from entering `date=ALFONCE`!

Reply:
Oh yes, adding error checking + logging